bin/lz4-asm

```diff
@@ -40,26 +60,28 @@ var argv = require('yargs')
 .epilog('If you want to show more info, refer to https://github.com/Cyan4973/lz4/blob/master/lz4_Frame_format.md')
 .argv
```

Comment:
Now, `argv` is a function call. Add `;`.

Reply:
Fixed